README.md

```diff
@@ -109,10 +114,11 @@ Compile and Deploy PerfKit Explorer
 
          cd ~/projects/PerfKitExplorer
 
-2. Modify the `app.yaml` file so that the instance class is appropriate for your
-   needs, and the application name matches the project id you created in the
-   'Create the App Engine project' step, and the version string is set
-   appropriately. For example:
+2. Modify the `app.yaml` file so that the
+   [instance class](https://cloud.google.com/appengine/docs/adminconsole/performancesettings)
+   is appropriate for your needs, and the application name matches the
+   project id you created in the 'Create the App Engine project' step,
+   and the version string is set appropriately. For example:
 
          application: perfkit-explorer-demo
          version: beta
```

Comment:
The minimum is the smallest instance class required.  As the number of points returned and pivoted grows, you'll need to up the instance class.  But that's in the 50-100k+ points per page range.